src/Mod/PartDesign/App/FeatureExtrude.h

```diff
@@ -41,13 +43,16 @@ class PartDesignExport FeatureExtrude : public ProfileBased
     App::PropertyEnumeration Type;
     App::PropertyLength      Length;
     App::PropertyLength      Length2;
+    App::PropertyAngle       TaperAngle;
+    App::PropertyAngle       TaperAngle2;
     App::PropertyBool        UseCustomVector;
     App::PropertyVector      Direction;
     App::PropertyBool        AlongSketchNormal;
     App::PropertyLength      Offset;
     App::PropertyLinkSub     ReferenceAxis;
 
     static App::PropertyQuantityConstraint::Constraints signedLengthConstraint;
+    App::PropertyAngle::Constraints floatAngle;
```

Comment:
Should be static as it's equal for all instances of pad/pocket.

Reply:
i know but could not get it to work. Either it works for me with MSVC then, I got a build failure of the CI. When the CI succeeds, I get a linking mistake. it cost me a while to realize that the definition of maxAngle causes this. When I use in maxAngle just a plain number, e.g.
"89.999"
then everything is fine, but as soon as I try to use
"90 - Base::toDegrees<double>(Precision::Angular()"
I got the problems.

I'll try later today if I find a solution that pleases the linker of MSVC and the CI compiler.

Reply:
OK, I found a solution, it is now a static member.